### ISSUE TITLE
How to process the HELEN into the required data format 

### ISSUE THREAD

**Issue by Beanocean** [FlYWMe/face-parsing#5]
Hi, FIYWMe

I want to do some experiments related to face parsing based on your code. 

I download the dataset Helen from the [URL](http://www.cs.wisc.edu/~lizhang/projects/face-parsing/). And there are two sizes of data provided. I want to know which resolution is used in your experiment.

Besides, would you please share your code that processes the raw dataset into the format required by this project.

Thanks

**Comment by FlYWMe** [comment]
@Beanocean Hi! The resized dataset is used in my experiment. The image list txt file is required and the data processing operations are defined in loadData.py file.

**Comment by Beanocean** [comment]
As I understand, I should first merge the separate binary masks into one mask which has 11 classes. 
And the list of images should  be like the follows:
```
<Path to RGB image>, <Path to Gray label mask>
```
Besides, you said 10146 images are applied for training, but there are only 2000 images in the training set. Do you generate the augmented training images offline or just use the augmentations in `torch.Transforms`?

**Comment by Beanocean** [comment]
The performance in Resnet18-UNet is only 14.5% in mIoU, and the stand UNet achieves 52.19% in mIoU. The hyperparameters are set as default in my training. And I use the script `train.py` to train the model. Do you have any recommendations to locate the bug? Is this possibly related to the PyTorch version?

**Comment by FlYWMe** [comment]
@Beanocean Yes, you should merge the separate binary masks into one mask which has 11 classes. Please check the label mask. The augmentation method include random crop, resize and horizontal flip. The PyTorch version doesn't matter, I guess.

**Comment by Beanocean** [comment]
@FIYWMe Thanks a lot for your reply. In my experiment, I use the following code to merge the masks.

``` python
import os
import cv2
import numpy as np

def main_cv2(indir="labels", outdir="merged"):
    for ii, x in enumerate(os.listdir(indir)):
        cur_dir = os.path.join(indir, x)
        label_file = os.path.join(cur_dir, "{}_lbl00.png".format(x))
        mask = cv2.imread(label_file, cv2.IMREAD_GRAYSCALE)
        mask[:] = 0  # set all pixel as zero

        for label_id in range(1, 11):
            label_file = os.path.join(
                cur_dir, "{}_lbl{:02d}.png".format(x, label_id))
            label_img  = cv2.imread(label_file, cv2.IMREAD_GRAYSCALE)
            index = label_img >= 128 # larger than 0.5 prob
            mask[index] = label_id

        if len(np.unique(mask).tolist()) > 11:
            print(np.unique(mask).tolist())

        out_path = os.path.join(outdir, "{}.png".format(x))
        cv2.imwrite(out_path, mask)
        if ii % 100 == 0:
            print(out_path)
```
Is it convinent to share your code or the merged masks?

**Comment by FlYWMe** [comment]
@Beanocean I took a look at the code. I think there is no problem. My code is on the server in the lab. I don’t know when I can go back. So, share it with you later：)
